deeppavlov_agent/core/connectors.py

```diff
@@ -1,27 +1,36 @@
 import asyncio
 from typing import Any, Callable, Dict, List
 from collections import defaultdict
+from logging import getLogger
+import os
 
+import sentry_sdk
 import aiohttp
 
 from .transport.base import ServiceGatewayConnectorBase
 
+logger = getLogger(__name__)
+sentry_sdk.init(os.getenv('DP_AGENT_SENTRY_DSN'))
+
 
 class HTTPConnector:
-    def __init__(self, session: aiohttp.ClientSession, url: str):
+    def __init__(self, session: aiohttp.ClientSession, url: str, timeout: int):
```

Comment:
Could be `float` used here instead of `int` for `timeout`?

Reply:
fixed